packages/scripts/src/multi-chain/BatchCollateralWhitelist.js

```diff
@@ -0,0 +1,75 @@
+// Simple script to initialize a whitelist on a new L2 chain by mapping the existing L1 whitelist. This script assumes
+// that the EOA unlocked on the L2 is currently the owner of both the AddressWhitelist and the Store contracts i.e
+// this is a fresh deployment of the L2 chain and ownership has not yet been transferred to the OracleSpoke.
+
+require("dotenv").config();
+
+const { getWeb3ByChainId } = require("@uma/common");
+
+const { getAbi, getAddress } = require("@uma/contracts-node");
+const { ZERO_ADDRESS } = require("@uma/common");
+
+const { fetchFullL1Whitelist, findL2TokenForL1Token } = require("./utils");
+const argv = require("minimist")(process.argv.slice(), { number: ["l1ChainId", "l2ChainId"] });
+
+async function run() {
+  const { l1ChainId, l2ChainId } = argv;
+
+  const l1Web3 = getWeb3ByChainId(l1ChainId);
+  const l2Web3 = getWeb3ByChainId(l2ChainId);
+  const l1Account = (await l1Web3.eth.getAccounts())[0];
+
+  console.log(`Running OVM Batch Collateral whitelister from ${l1ChainId}->${l2ChainId} with`);
```

Comment:
I wouldn't name this just OVM batch whitelister, since it should work on other L2s right?

Reply:
ye, that's a mistake. when I started it was OVM spesific but I was able to generalize it.